content.js

```diff
@@ -74,93 +74,104 @@ window.addEventListener("message", function (event) {
-                $(card).find("[class*=_footerChildSection]").eq(1).append("<span class=\"ghx-field gh-labels-in-jira\" data-tooltip=\"0 pull requests\">" + NO_PR_ICON + "</span>");
-        } else {
-            if ($(card).find(".gh-labels-in-jira").length == 0)
-                $(card).find("[class*=_footerChildSection]").eq(1).append("<span class=\"ghx-field gh-labels-in-jira\" style=\"cursor:pointer;\" data-tooltip=\"" + data.detail[0].pullRequests.length + " pull request(s)\" onclick=\"event.stopPropagation();window.postMessage({ type: 'refreshPRs', params: { issueKey: '" + $(card).attr("id") + "'} }, '*');\">" + PR_ICON + "</span>");
-
-            var prIcon = $(card).find(".ghx-field .gh-labels-in-jira");
-            $(card).find("[class*=_content]").append("<div class=\"gh-labels-in-jira-wrapper\"></div>");
-            var wrapper = $(card).find(".gh-labels-in-jira-wrapper");
-
-            var cardId = $(card).attr("id");
-
-            var hasHeader = false;
-
-            $.each(data.detail[0].pullRequests, function () {
-                var prid = this.id;
-                var owner = this.url.split("/")[3];
-                var repo = this.url.split("/")[4];
+    // As of 11/02/23, there is no public facing API to fetch pull requests associated with issues. 
+    // but there is an internal one, which is volatile and subject to change without notice 
+    // See this Jira issue here https://jira.atlassian.com/browse/JSWCLOUD-16901
+    // This internal endpoint takes in the issueId, which is not longer readily available in the UI DOM.
+    // so use the issue key (i.e. MOAPT-1234) to fetch the issue Id from a more stable API 
+    // https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issues/#api-rest-api-3-issue-issueidorkey-get
 
-                // Hide both declined and merged PRs if HIDE_CLOSED_PRS
-                const hideLabels = this.status == "DECLINED" && HIDE_CLOSED_PRS || this.status == "MERGED" && HIDE_MERGED_PRS;
+    var cardId = $(card).attr("id").replace("card-", "");
 
-                // Apply the header, only if we have at least one label
-                if (!hideLabels && hasHeader === false) {
-                    hasHeader = true;
-                    $(wrapper).append("<div class=\"pr-heading\">Pull Requests</div>");
-                }
-                
-                if (!hideLabels) {
-                    var pullRequestNode = document.createElement("div");
-                    pullRequestNode.classList.add("pullRequestNode");
-                    pullRequestNode.setAttribute("data-ticket-pull-id", cardId.replace("card-", "") + "-" + prid.replace("#", ""));
+    $.getJSON("https://" + JIRA_HOSTNAME + "/rest/api/3/issue/" + cardId + "?fields=id", function(data) {
+        var issueId = data?.id;
+    
+        $.getJSON("https://" + JIRA_HOSTNAME + "/rest/dev-status/latest/issue/detail?issueId=" + issueId + "&applicationType=GitHub&dataType=pullrequest", function (data) {
```

Comment:
Everything from this line down is just me indenting the subsequent API call. The only real change here is the additional, above API call to fetch the issue ID